### DISCUSSION TITLE
Is it possible to set the default change frequency?

### DISCUSSION THREAD

**Discussion by jonathands** [spatie/laravel-sitemap#531] [General]
I understand we  can change the changeFreq on a per url basis, but since I would just like to let the crawler grab all the urls, I don't have them all available (something like 20.000).

Is it possible to change the default change frequency from daily to other values?

**Comment by mhd-fettah** [comment]
you dont need the changefrequency . 
>Google ignores priority and changefreq values. 
https://developers.google.com/search/docs/crawling-indexing/sitemaps/build-sitemap
